src/ci/tested_versions_utils.py

```diff
@@ -151,7 +152,9 @@ def parse_version(version: str) -> Union[SemanticVersion, NonSemanticVersion]:
 
 @dataclass(frozen=True)
 class TestedVersions:
-    versions: List[Union[SemanticVersion, NonSemanticVersion]]
+    versions: List[Union[SemanticVersion, NonSemanticVersion]] = attrib(
```

Comment:
Notice this warning from the documentation
![image](https://user-images.githubusercontent.com/49228804/180209791-a6f5b42e-145c-4e50-9146-ab5b727e9530.png)

Reply:
good catch

Reply:
Done in 730ed4f265e9fb3d097c8032be504507ee0bcbfd